src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementService.java

```diff
@@ -541,17 +541,22 @@ public void completeTask(String taskId, AccessControlResponse accessControlRespo
 
         //Lock & update Task
         TaskResource task = findByIdAndObtainLock(taskId);
-        taskHasCompleted = task.getState() == CFTTaskState.COMPLETED;
+        CFTTaskState state = task.getState();
+        taskHasCompleted = state != null
```

Comment:
New line of code and is partially covered.  May need to include terminationReason in test where cftTask.state = COMPLETED.